pandas/core/groupby/generic.py

```diff
@@ -266,7 +266,11 @@ def aggregate(self, func=None, *args, engine=None, engine_kwargs=None, **kwargs)
             func = maybe_mangle_lambdas(func)
             ret = self._aggregate_multiple_funcs(func)
             if relabeling:
-                ret.columns = columns
+                # error: Incompatible types in assignment (expression has type
+                # "Optional[List[str]]", variable has type "Index")
+                ret.columns = columns  # type: ignore[assignment]
+            return ret
+
```

Comment:
you could outdent (and remove the else) here to make it more obvious

Reply:
considered that, but this mirrors the layout of the DataFrameGroupBy method and im holding out hope of sharing more of that